sample/src/androidTest/java/com/schibsted/spain/barista/sample/RecyclerViewTest.java

```diff
@@ -26,4 +27,10 @@ public void checkClickRecyclerViewItem_byPosition_atThree() {
     clickRecyclerViewItem(R.id.recycler, 3);
     assertDisplayed("Raspberry");
   }
+
+  @Test
+  public void checkScrollToRecyclerViewItem_byPosition_atThree() {
+    scrollTo(R.id.recycler, 3);
```

Comment:
yes 😅 
If you want, I could add more elements to this recycle to test the functionality